vmdb/app/models/vmdb_database_connection.rb

```diff
@@ -17,27 +29,67 @@ class VmdbDatabaseConnection < ActsAsArModel
   virtual_belongs_to :miq_worker
 
   virtual_column :pid, :type => :integer
+  virtual_column :blocked_by, :type => :integer
+
+  attr_reader :vmdb_database_id
+
+  def vmdb_database_id
+    @vmdb_database_id ||= self.class.vmdb_database.id
+  end
 
-  def initialize(values = {})
-    values[:vmdb_database] ||= self.class.vmdb_database
-    super(values)
+  def address
+    client_addr
   end
```

Comment:
`alias_attribute` also defines all the extra methods like `blah_changed?`, etc.  We weren't implementing those before with `ActsAsArModel`, so presumably the system doesn't need those methods for these attributes.  I personally like defining the proxy method because it's really clear what the method is doing.  It's not so clear what the side effects of calling `alias_attribute` are (unless you read the Rails docs).

Anyway, if we start needing those extra methods, we can easily switch these.